### PR TITLE
[DNM] Translations: Updated French/German and adding es, it, ja, ko-kr, pt-br, zh-cn, zh-tw

### LINKED ISSUE #376: Strings and messages for Preview functionality should be rephrase
**Problem:**
Current English messages in build for Preview functionality are misleading and should be rephrase to more meaningful or informative sentences.

**Proposed Solution:**
Here is some suggestion on how to rephrase strings for Preview harvested from /src/extension/_locales/en/messages.json
* $1 files successfully previewed. > Preview of $1 files successfully generated. OR
* $1 of $2 files previewed... > Preview $1 of $2 files generated...
* 1 file successfully previewed. > Preview of this file successfully generated.
* Are you sure you want to preview $1 files? > Are you sure you want to generate a preview for $1 files?
* Are you sure you want to preview 1 file? > Are you sure you want to generate a preview for this file?
* Failed to preview $1 files: > Failed to generate preview of $1 files:
* Failed to preview 1 file: > Failed to generate preview of 1 file:
* File must be previewed first > A preview of the file must be generated first
* Previewed: > Preview generated
* Previewing failed. Must be Google document or spreadsheet. > Preview generation failed. Must be Google document or spreadsheet.
* Previewing failed. Please try again later. > Preview generation failed. Please try again later.
* Pro tip: Press ⌘ + S before previewing to see the latest changes immediately... > Pro tip: Press ⌘ + S before generating preview to see the latest changes immediately
* Pro tip: Press Ctrl + S before previewing to see the latest changes immediately... > Pro tip: Press Ctrl + S before generating preview to see the latest changes immediately...

### LINKED ISSUE #381: [i18n] Language fallback always on English even if other supported languages are more prioritized in Preferred languages list
**Description**
In case browser Preferred languages list contain more preferred unsupported language before supported one, then Language fallback mechanism will load extension on English even if it not mentioned in Preferred languages list.

**To Reproduce**
1. Open chrome and go to Setting > Languages (chrome://settings/languages)
2. Set-up language priority as follow Italian > German > French
3. Go to Google Drive repository that is added as project for extension
4. Click on extension icon to initialize extension panel

**Expected behavior**
Extension panel is appear and loaded on German

**Actual behavior**
Extension panel is appear and loaded on English

**Screenshots**
Language_fallback

### PR DESCRIPTION
**Pull request by rgeraghty**
Please note our system sorts the translated files, as this is first commit, French and German have large DIFF, but subsequent ones will be same order, with smaller diff.

Other issues addressed in this branch/PR:
- Fix #376
- Fix #381

### CODE REVIEW

**Comment by rofe** [comment]
@rgeraghty it's difficult for me to see the actual diffs for existing languages. Is there a way you could restore the original order of strings for `de` and `fr`?

File: src/extension/_locales/fr/messages.json [hunk 1]
```diff
@@ -1,501 +1,503 @@
...
   },
-  "plugins_loading": {
-    "message": "Chargement..."
+  "bulk_confirm_empty": {
+    "message": "Aucun fichier sélectionné"
   },
-  "plugins_empty": {
-    "message": "Aucune action disponible pour cette page"
+  "bulk_confirm_preview_multiple": {
+    "message": "Voulez-vous vraiment prévisualiser les fichiers $1 ?"
```
Comment (rofe) [inline src/extension/_locales/fr/messages.json:22]: This doesn't look right: `$1` is an integer (number of files), not a string (file names). Please check translations for other languages as well.

Comment (rgeraghty) [reply src/extension/_locales/fr/messages.json:22]: @rofe this is just related to localized files are sorted, matching English is
```
  "bulk_confirm_preview_multiple": {
    "message": "Are you sure you want to preview $1 files?"
  },
```

Comment (rofe) [reply src/extension/_locales/fr/messages.json:22]: @rgeraghty I meant the translation that is wrong (in multiple languages: 
`$1` is a number, so for an English string:
`Are you sure you want to preview 7 files?`
you'd get the following in French:
`Voulez-vous vraiment prévisualiser les fichiers 7 ?`
while the expected output is:
`Voulez-vous vraiment prévisualiser 7 fichiers ?`

Comment (rgeraghty) [reply src/extension/_locales/fr/messages.json:22]: Thanks @rofe I see, we are correcting

**Comment by rgeraghty** [comment]
@rofe while checking the "Are you sure you want to preview $1 files?" string you mention above, there are a few other things we need to check (such as few occurrences of 1$ instead of $1).

We are checking these now with vendor and I'll update translations again tomorrow, to improve them,

File: src/extension/_locales/fr/messages.json (hunk 1 above)
Comment (rgeraghty) [reply src/extension/_locales/fr/messages.json]: @rofe the translators has gone over these and corrected, example for French and German:
```
Voulez-vous vraiment prévisualiser $1 fichiers ?

Möchten Sie wirklich eine Vorschau von $1 Dateien anzeigen?
```

**Comment by rgeraghty** [comment]
Hi @rofe the translators and gone through the files and made some improvements in a number of areas, so I re-did the diff (due to sort change), here is the latest:

[fr_differences.txt](https://github.com/adobe/helix-sidekick-extension/files/11548235/fr_differences.txt)
[de_differences.txt](https://github.com/adobe/helix-sidekick-extension/files/11548236/de_differences.txt)

Please let us know any issue, hopefully this is ready now for merge,

File: src/extension/_locales/fr/messages.json (hunk 1 above)
Comment (rgeraghty) [reply src/extension/_locales/fr/messages.json]: @rofe we have passed your comment to vendor/translators to update

Comment (rgeraghty) [reply src/extension/_locales/fr/messages.json]: @rofe we have update the translations, per your comment for 'generating the preview'. The translators update all locales, not just German.

**Comment by rgeraghty** [comment]
@rofe with the translation updated for 'generating the preview' please find here latest DIFF:
[fr_differences.txt](https://github.com/adobe/helix-sidekick-extension/files/11566910/fr_differences.txt)
[de_differences.txt](https://github.com/adobe/helix-sidekick-extension/files/11566911/de_differences.txt)

**Review by rofe — CHANGES_REQUESTED** [review]
@rgeraghty some more observations...

**Review by kptdobe — APPROVED** [review]
FR ok for me. Just one minor subjective detail...

**Review by kptdobe — APPROVED** [review]
FR ok for me. Just one minor subjective detail...

**Comment by kptdobe** [comment]
Could we simply sort the files first in the `main` branch ? 
Review might be easier if the files would be sorted first in `main` and then do the updated translations... at the moment, really hard to see the changes.

**Comment by rgeraghty** [comment]
I've marked this do not merge, as...
- encountered conflicts trying to rebase related to change to lang-support, and don't want to mess up/have to reset this PR and lose comments
- I'll now on PTO, and won't be here when translations come back, better for Masa to create a clean PR, for the updated translations, directly to lang-support

**Pull request closed without merging**